clippy_lints/src/neg_cmp_op_on_partial_ord.rs

```diff
@@ -79,7 +79,7 @@ impl<'tcx> LateLintPass<'tcx> for NoNegCompOpForPartialOrd {
                         cx,
                         NEG_CMP_OP_ON_PARTIAL_ORD,
                         expr.span,
-                        "The use of negated comparison operators on partially ordered \
+                        "the use of negated comparison operators on partially ordered \
```

Comment:
this one still has sentences and punctuation but had the initial capitalization removed. I'm not sure which direction to go on this one but we should be consistent.

Reply:
I was also not sure... The [guide](https://rustc-dev-guide.rust-lang.org/diagnostics.html) says `The text should be matter of fact and avoid capitalization and periods, unless multiple sentences are needed` which why I left it mostly untouched.

Reply:
Makes sense, then we should continue to capitalize the first letter given that this is the "unless multiple sentences are needed" case.

```suggestion
                        "The use of negated comparison operators on partially ordered \
```

Reply:
I think converting all multi-sentence warnings into uber-long single-sentence warnings would probably hurt readability too much...